### PR TITLE
Big pr (sorry)

### PR DESCRIPTION
**Pull request by OskarDamkjaer**
Add a /notify route. Rename Numbers => numberList. Made some changes to the css with the padding since adding paddin 2% globally caused the site to be over 100% for me at least :). Added eslint, but no hook since there are very many errors.

### CODE REVIEW

**Comment by OskarDamkjaer** [comment]
ping @wanecek :D

File: package.json [hunk 1]
```diff
@@ -5,13 +5,24 @@
   "main": "app.js",
   "scripts": {
     "start": "node app.js",
-    "setup": "yarn && cd front-end/ && yarn && yarn build && cd .. && node app.js"
+    "setup": "yarn && cd front-end/ && yarn && yarn build && cd .. && node app.js",
+    "lint": "./node_modules/.bin/eslint '*.js' --fix && ./node_modules/.bin/eslint './front-end/src/**/*.js' --fix"
```
Comment (wanecek) [inline package.json]: It should work to just refer to `eslint` directly, since it's in our dependencies :)

File: package.json [hunk 2]
```diff
@@ -5,13 +5,24 @@
...
-    "cors": "^2.8.4",
     "express": "^4.16.4"
+  },
+  "devDependencies": {
+    "eslint": "5.6.0",
+    "eslint-config-airbnb": "^17.1.0",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-jsx-a11y": "^6.1.2",
+    "eslint-plugin-react": "^7.11.1",
+    "husky": "^1.1.3"
```
Comment (wanecek) [inline package.json]: Husky is added, but I don't see it being used. Maybe worth implementing it, or adding it later? :)

File: front-end/src/utils/NotifyMe.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import React from 'react'
```
Comment (wanecek) [inline front-end/src/utils/NotifyMe.js]: Run this file through prettier as well :)

File: front-end/src/index.css [hunk 4]
```diff
@@ -16,13 +17,6 @@ body {
   -moz-osx-font-smoothing: grayscale;
   background-color: #f9eae1;
   color: #424b54;
-  padding: 2%;
```
Comment (wanecek) [inline front-end/src/index.css]: I assume that these changes are intentional? :)

File: front-end/src/Routes/AdminRoute.js [hunk 5]
```diff
@@ -1,16 +1,17 @@
...
-import AdminView from '../AdminBar/AdminView'
-import { AppContainer } from '../utils/styles'
+import React from 'react';
+import NumberContainer from '../Numbers/NumberContainer';
+import NumberListFetcher from '../Numbers/NumberListFetcher';
+import AdminView from '../AdminBar/AdminView';
+import { AppContainer } from '../utils/styles';
 
-const AdminRoute = () => (
+const NotifyRoute = () => (
```
Comment (wanecek) [inline front-end/src/Routes/AdminRoute.js]: Shouldn't this be called `AdminRoute`? :thinking:

**Comment by wanecek** [comment]
Really cool, nice work! \o/ Added a few comments, mostly nitpicks. 
You might want to make the UI a bit larger for the controls on the notify page though, to make it mobile-friendly (I assume that's where it will be used in the majority of the cases) :slightly_smiling_face: 

My initial idea was that these controls would be added to the main view, available through some sort of control/toggle (and probably somehow hidden on the big tv, e.g. through a query parameter), but now that I see your solution I'm not so sure anymore :smile: Let's merge this once you feel ready, and we can iterate in the future (although changing URL _might_ confuse users a bit, but w/e)

File: front-end/src/AdminBar/adminStyles.js [hunk 6]
```diff
@@ -13,8 +14,10 @@ export const AdminPanel = styled.div`
     grid-template-areas:
      "clear undo send"
      "currentInput currentInput currentInput";
+    width: 100%; 
+    padding: 0%;
```
Comment (wanecek) [inline front-end/src/AdminBar/adminStyles.js]: The nit of all nits: It's quite common to prefer unitless values for `0`, but I also hate myself a tiny bit for commenting on it

File: front-end/src/utils/NotifyMe.js [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+import { ColoredButton } from '../AdminBar/adminStyles'
+import { NotifyContainer } from './styles'
+
+class NotifyMe extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {input: '', acceptsNotis: false, sentNotis: false, waitingFor: null, hide: false}
```
Comment (wanecek) [inline front-end/src/utils/NotifyMe.js]: WDYT about using the english term here, i.e. `acceptsNotification`? :)

File: package.json (hunk 1 above)
Comment (OskarDamkjaer) [reply package.json]: Oh, you are right! You learn something new every day :)

File: package.json (hunk 2 above)
Comment (OskarDamkjaer) [reply package.json]: I added it but eslint has a lot of errors, so for now there is no pre commit hook! I forgot to remove it O:)

File: front-end/src/Routes/AdminRoute.js (hunk 5 above)
Comment (OskarDamkjaer) [reply front-end/src/Routes/AdminRoute.js]: Whooops! thanks!

File: front-end/src/AdminBar/adminStyles.js (hunk 6 above)
Comment (OskarDamkjaer) [reply front-end/src/AdminBar/adminStyles.js]: I do prefer them as well! No worries 💯

File: front-end/src/utils/NotifyMe.js (hunk 7 above)
Comment (OskarDamkjaer) [reply front-end/src/utils/NotifyMe.js]: That'd be better, since the rest of the code is english

**Comment by OskarDamkjaer** [comment]
Yes the UI for notify was not very polished yet! I wasn't sure how it would be implemented in the best way so i put it in another route. Perhaps a queryparm would be better? Now use only prettier and added a hook aswell! :D

**Pull request merged**